.asf.yaml

```diff
@@ -35,7 +35,7 @@ github:
       required_status_checks:
         strict: true
         contexts:
-          - Build and Test
+          - Required
```

Comment:
You can simply remove the original one, that prevents your PR being merged

Reply:
I have removed the matrix in the `Build and Test` job.

Reply:
You still need to restore the original one `Build and Test` back